README.adoc

```diff
@@ -10,6 +10,57 @@ image:https://goreportcard.com/badge/github.com/fabric8-services/fabric8-env[Go
 image:https://godoc.org/github.com/fabric8-services/fabric8-env?status.png[GoDoc,link="https://godoc.org/github.com/fabric8-services/fabric8-env"]
 image:https://codecov.io/gh/fabric8-services/fabric8-env/branch/master/graph/badge.svg[Codecov.io,link="https://codecov.io/gh/fabric8-services/fabric8-env"]
 
-Environment service takes care of managing environment.
+Swagger API: http://swagger.goa.design/?url=github.com%2Ffabric8-services%2Ffabric8-env%2Fdesign[link]
+
+=== Developer Guide
+
+==== Prerequisites
+
+go (>=1.8), git, mercurial, make, dep
+
+==== Make commands
+
+===== Build project
+
+**Generate Resources:** generate goa, sqlbindata, swagger related resources.
+----
+make generate
+----
+
+NOTE: `./app/`, `./swagger/`, `sqlbindata*.go` under `./migration/**` are generated and should not be edited.
+
+**Build Project:** generate resources, build the project, create executable at `./bin/env`.
+----
+make build
+----
+
+**Clean Project:** clean generated resources and vender, tmp, bin directories.
+----
+make clean
+----
+
+===== Run service
+
+**Run service on localhost:** start service on localhost. Check status at - `http://localhost:8080/api/status`.
+----
+export F8_CONFIG_FILE_PATH="config.yaml"
+docker-compose up -d db
+./bin/env
```

Comment:
Do you think that it would be possible to create a separated target for that? eg. `make dev`?

Reply:
Exactly. We already have `make dev`. The PR you mentioned fixes the default config so `make dev` will work - https://github.com/fabric8-services/fabric8-env/pull/13
But `make dev` not always work fine because of inotify limit problem in docker, so, while we can mention `make dev` the more long way of running service is more reliable.